query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -0,0 +1,204 @@
+/**
+ * Run with: `node --experimental-wasm-modules ./example.js`
+ * on Node.js 18+.
+ */
+import { webcrypto } from "node:crypto";
+import * as qe from "./qe";
+
+import pgDriver from "pg";
+import * as prismaPg from "@prisma/adapter-pg";
+import { bindAdapter, DriverAdapter } from "@prisma/driver-adapter-utils";
+
+import { recording } from "./recording";
+import prismaQueries from "../bench/queries.json";
+
+import { run, bench, group, baseline } from "mitata";
+
+import { QueryEngine as WasmBaseline } from "query-engine-wasm-baseline";
+import { QueryEngine as WasmLatest } from "query-engine-wasm-latest";
+
+(global as any).crypto = webcrypto;
+
+async function main(): Promise<void> {
+  // read the prisma schema from stdin
+  const datamodel = await new Promise<string>((resolve, reject) => {
```

Comment:
Can we just read the schema from the file directly instead of stdin? That would also save us the need for `bench.sh` script, i think